### ISSUE TITLE
`object-shorthand` warns on computed properties

### ISSUE THREAD

**Issue by sodatea** [eslint/eslint#6071]
**What version of ESLint are you using?**
v2.9.0
**What parser (default, Babel-ESLint, etc.) are you using?**
Espree
**Please show your full configuration:**
```yaml
parserOptions:
  ecmaVersion: 7
rules:
  object-shorthand: [error, always]
```
**What did you do? Please include the actual source code causing the issue.**
```javascript
const className = 'aaa';

const obj = {
  [className]: className
};

console.log(obj);
```
**What did you expect to happen?**
No warnings, no errors.

**What actually happened? Please include the actual, raw output from ESLint.**
```
$ eslint object-shorthand.js

/Users/soda/Tests/eslint/object-shorthand.js
  4:3  error  Expected property shorthand  object-shorthand

✖ 1 problem (1 error, 0 warnings)
```

**Comment by michaelficarra** [comment]
Duplicate of #6015.

**Comment by mysticatea** [comment]
Closing as this have been fixed already.
Please wait for next release.
